public/v2/apps/nextcloud.json

```diff
@@ -1,31 +1,83 @@
 {
     "captainVersion": "2",
     "documentation": "Taken from https://hub.docker.com/_/nextcloud",
-    "displayName": "",
+    "displayName": "nextcloud",
     "description": "Nextcloud is a suite of client-server software for creating and using file hosting services",
     "dockerCompose": {
         "version": "3.3",
         "services": {
+            "$$cap_appname-db": {
+                "documentation": "Taken from https://hub.docker.com/_/mariadb",
+                "image": "mariadb:$$cap_mariadb_version",
+                "notExposeAsWebApp": "true",
+                "volumes": [
+                    "$$cap_appname-db-data:/var/lib/mysql"
+                ],
+                "restart": "always",
+                "environment": {
+                    "MYSQL_ROOT_PASSWORD": "$$cap_db_pass",
+                    "MYSQL_DATABASE": "nextcloud",
+                    "MYSQL_USER": "$$cap_db_user",
+                    "MYSQL_PASSWORD": "$$cap_db_pass"
+                }
+            },
             "$$cap_appname": {
+                "depends_on": [
+                    "$$cap_appname-db"
+                ],
+                "documentation": "Taken from https://hub.docker.com/_/nextcloud",
                 "image": "nextcloud:$$cap_nextcloud_version",
                 "volumes": [
-                    "$$cap_appname-data:/var/www/html"
+                    "$$cap_appname-website:/var/www/html",
+                    "$$cap_appname-apps:/var/www/html/custom_apps",
+                    "$$cap_appname-config:/var/www/html/config",
+                    "$$cap_appname-data:/var/www/html/data"
```

Comment:
You don't need to define the last 3 volumes. Only the top one would be enough.

Reply:
agreed, fixed